pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -702,67 +730,68 @@ func TestReconcile_CustomTask(t *testing.T) {
 	const pipelineRunName = "test-pipelinerun"
 	const pipelineTaskName = "custom-task"
 	const namespace = "namespace"
+
+	simpleCustomTaskPRYAML := `metadata:
```

Comment:
This and `simpleCustomTaskWantRunYAML` are here so that we can use the same input `PipelineRun` and expected `Run` for test cases with different `embedded-status` values without duplicating everything.